include/constants/item_config.h

```diff
@@ -0,0 +1,33 @@
+#ifndef GUARD_CONSTANTS_ITEM_CONFIG_H
+#define GUARD_CONSTANTS_ITEM_CONFIG_H
+
+// Used by other branches to communicate with each other.
+#define ITEM_EXPANSION
+
+#ifndef GEN_3
+#define GEN_3 0
+#define GEN_4 1
+#define GEN_5 2
+#define GEN_6 3
+#define GEN_7 4
+#define GEN_8 5
+#endif
+
+// Item config
+#define I_SHINY_CHARM_REROLLS   3       // Amount of re-rolls if the player has the Shiny Charm. Set to 0 to disable Shiny Charm's effects.
+#define I_KEY_FOSSILS           GEN_7   // In Gen4+, all Gen 3 fossils became regular items.
+#define I_KEY_ESCAPE_ROPE       GEN_7   // In Gen8, Escape Rope became a Key Item.
```

Comment:
could you put a note here saying that setting this to `GEN_8` will make it be sold for free in shops

![](https://i.imgur.com/Sb604J4.png)

not a bug but it should be noted